Xcode 11.4 beta2: `_bit_scan_forward` redefinition

Vc version / revision | Operating System | Compiler & Version | Compiler Flags | Assembler & Version | CPU
----------------------|------------------|--------------------|----------------|---------------------|----
           1.4.1           |         Mac         |          Xcode 11.4 beta 2 (11N123k)         |    ROOT's            |                     |

## Testcase
```sh
clang++ -std=c++11 -fsyntax-only -x c++-header -I /Users/sftnight/build/jenkins/night/LABEL/mac10beta/SPEC/cxx17/V/6-20/build/include/ /Users/sftnight/build/jenkins/night/LABEL/mac10beta/SPEC/cxx17/V/6-20/build/include/Vc/Vc
```

## Actual Results
```
In file included from /Users/sftnight/build/jenkins/night/LABEL/mac10beta/SPEC/cxx17/V/6-20/build/include/Vc/Vc:30:
In file included from /Users/sftnight/build/jenkins/night/LABEL/mac10beta/SPEC/cxx17/V/6-20/build/include/Vc/vector.h:35:
In file included from /Users/sftnight/build/jenkins/night/LABEL/mac10beta/SPEC/cxx17/V/6-20/build/include/Vc/sse/vector.h:496:
In file included from /Users/sftnight/build/jenkins/night/LABEL/mac10beta/SPEC/cxx17/V/6-20/build/include/Vc/scalar/../common/../sse/vector.tcc:30:
/Users/sftnight/build/jenkins/night/LABEL/mac10beta/SPEC/cxx17/V/6-20/build/include/Vc/scalar/../common/../sse/../common/bitscanintrinsics.h:37:13: warning: '_bit_scan_forward' macro redefined [-Wmacro-redefined]
#    define _bit_scan_forward(x) __builtin_ctz(x)
            ^
/Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/11.0.3/include/ia32intrin.h:72:9: note: previous definition is here
#define _bit_scan_forward(A) __bsfd((A))
        ^
```
Same for `_bit_scan_reverse`

## Expected Results
No warning!

--- a/Vc/common/bitscanintrinsics.h
+++ b/Vc/common/bitscanintrinsics.h
@@ -29,11 +29,8 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define VC_COMMON_BITSCANINTRINSICS_H_
 
 #if defined(Vc_GCC) || defined(Vc_CLANG) || defined(Vc_APPLECLANG)
-#  if Vc_GCC >= 0x40500
-     // GCC 4.5.0 introduced _bit_scan_forward / _bit_scan_reverse
-#    include <x86intrin.h>
-#  else
-     // GCC <= 4.4 and clang have x86intrin.h, but not the required functions
+#include <x86intrin.h>
+#  ifndef _bit_scan_forward
 #    define _bit_scan_forward(x) __builtin_ctz(x)
 #include "macros.h"
 static Vc_ALWAYS_INLINE Vc_CONST int _Vc_bit_scan_reverse_asm(unsigned int x) {

